config.yaml

```
@@ -12,5 +12,5 @@ options:
     description: Which mode out of `development` or `production` the backend server should use
   secure-cookies:
     type: boolean
-    default: false
+    default: true
```

I should leave this as `false`.  We in the past had troubles with this setting
```suggestion
    default: false
```